h2o-automl/src/main/java/hex/schemas/TargetEncoderModelOutputV3.java

```diff
@@ -0,0 +1,7 @@
+package hex.schemas;
+
+import ai.h2o.automl.targetencoding.TargetEncoderModel;
+import water.api.schemas3.ModelOutputSchemaV3;
+
+public class TargetEncoderModelOutputV3 extends ModelOutputSchemaV3<TargetEncoderModel.TargetEncoderOutput, TargetEncoderModelOutputV3> {
```

Comment:
empty output? is this final?

Reply:
Added only prior_mean and renamed ModelOutput to make proper implicit attribute mapping possible. 

Note (to self): We should prbably provide proper model summary for TargetEncoderModel.

Reply:
@deil87 Important question: Please have at what goes into model summary and model output in general and let me know if there is something more to expose and I'll do it.

Reply:
@Pscheidl As we discussed in Slack... prior mean is probably the only reasonable thing for now. Encoding map might be too big.... maybe truncated version is ok but doesn't make a lot of sense. Info about Blending could be exposed through exposure of model's parameters. We can provide some helpful summary but we will need to compute these statistics e.g. number of instances which were imputed with prior mean; how many categories were represented with 1 instance, 2 instances etc.